march_gazebo_plugins/include/stairs_controller.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Project March.
+
+#include <obstacle_controller.h>
+
+#ifndef MARCH_GAZEBO_PLUGINS_STAIRS_CONTROLLER_H
+#define MARCH_GAZEBO_PLUGINS_STAIRS_CONTROLLER_H
+
+namespace gazebo
+{
+class StairsController : public ObstacleController
+{
+public:
+  explicit StairsController(physics::ModelPtr model);
+  void update(ignition::math::v4::Vector3<double>& torque_all, ignition::math::v4::Vector3<double>& torque_stable);
```

Comment:
Add `override` to make clear that this method is overridden
```suggestion
  void update(ignition::math::v4::Vector3<double>& torque_all, ignition::math::v4::Vector3<double>& torque_stable) override;
```